### ISSUE TITLE
printf / sscanf format string validation

### ISSUE THREAD

**Issue by rk** [kalessil/phpinspectionsea#171]
I noticed 2 false positives today:

```php
// Based on: http://php.net/manual/en/function.sscanf.php#63526
// "Pattern seems to be invalid"
fscanf($handle, '%s %s %s [%[^]]] "%s %s %[^"]" %d %d "%[^"]" "%[^"]"',
	$log['ip'],
	$log['client'],
	$log['user'],
	$log['time'],
	$log['method'],
	$log['uri'],
	$log['prot'],
	$log['code'],
	$log['bytes'],
	$log['ref'],
	$log['agent']
);
```

And:

```php
$i=[1,2,3];
printf('%d %d %d', ...$i); // Amount of expected parameters is 4
```

**Comment by kalessil** [comment]
Thank you for reporting @rk, good catch =)

**Comment by rk** [comment]
You're welcome. Thanks for such a great tool!
